### PR TITLE
EOS-19505-Provisioner destroy command doesn't clean up all the rpms

### PR DESCRIPTION
**Pull request by tanujashinde0405**
Signed-off-by: Tanuja Shinde <tanuja.shinde@seagate.com>

### CODE REVIEW

File: devops/ci/multi_node_deploy/3node_destroy.jenkinsfile [hunk 1]
```diff
@@ -123,7 +123,8 @@ node {
             stage("${remote.name}: Uninstall rpms") {
                 sshCommand remote: remote, command: """
                     yum erase -y cortx-prvsnr cortx-prvsnr-cli      # Cortx Provisioner packages
```
Comment (ypise) [inline devops/ci/multi_node_deploy/3node_destroy.jenkinsfile:125]: Should not be necessary to erase if our destroy API has worked well.

File: srv/components/system/firewall/teardown.sls [hunk 2]
```diff
@@ -67,3 +67,8 @@ Refresh firewalld:
 Delete firewall checkpoint flag:
   file.absent:
     - name: /opt/seagate/cortx_configs/provisioner_generated/{{ grains['id'] }}.firewall
+
+Stop and disable firewalld service:
```
Comment (ypise) [inline srv/components/system/firewall/teardown.sls:71]: Don't stop the service.
Check with @SomwanshiAnjali for the best approach.

File: srv/components/system/storage/glusterfs/teardown/cache_remove.sls [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+Remove glusterd cache:
+  file.absent:
+    name: /var/lib/glusterd
```
Comment (ypise) [inline srv/components/system/storage/glusterfs/teardown/cache_remove.sls:20]: Check if there's something under `/var/cache` for gluster to be cleaned.

**Review by ypise — COMMENTED** [review]
@mazinamdar , @SomwanshiAnjali 
Check for gluster and firewall teardown if the states are in right order.

File: srv/components/system/storage/glusterfs/teardown/cache_remove.sls (hunk 3 above)
Comment (tanujashinde0405) [reply srv/components/system/storage/glusterfs/teardown/cache_remove.sls:20]: Nothing under /var/cache found related to gluster to be cleaned

**Review by ypise — APPROVED** [review]

**Review by ypise — APPROVED** [review]

**Review by mazinamdar — APPROVED** [review]
looks good to me

**Pull request merged**